smsdk/smsdk_entities/machine/machine_v1.py

```diff
@@ -64,3 +65,39 @@ def get_type_from_machine_name(self, machine_source, *args, **kwargs):
             ):
                 machine_type = record["type"]
         return machine_type
+
+    def modify_query_params(self, url, kwargs):
+        where = []
+        order_by = []
+        params = {}
+        for key, value in kwargs.items():
+            where_query = {}
+            orderby_query = {}
+            select_query = {}
+            if not key.startswith("_"):
+                where_query["name"] = key
+                where_query["value"] = value
+                where.append(where_query)
+            elif key == "_order_by":
+                orderby_query["name"] = (
+                    value.replace("-", "") if value.startswith("-") else value
+                )
+                orderby_query["order"] = "desc" if value.startswith("-") else "asc"
+                order_by.append(orderby_query)
+
+        select = [{"name": i} for i in eval(kwargs.get("_only", "[]"))]
```

Comment:
Why an `eval`? Add a comment about its usage.

Reply:
Just to get the original datatype of `kwargs.get("_only")`, I will add the comment as well.